conjure_oxide/tests/integration/basic/abs/03-nested/input.eprime

```diff
@@ -0,0 +1,7 @@
+language ESSENCE' 1.0
+
+find x,y,z: int(-5..2)
```

Comment:
this might already be testing the -2..3 case I was worried about, but still adding another one would make me happier. just `find x : int(-2..3) such that |x| = 1` would be enough.

Reply:
Apart from an unrelated typo, this test works